beekeeper-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/filter/IcebergTableListenerEventFilter.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (C) 2019-2024 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.scheduler.apiary.filter;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.stereotype.Component;
+
+import com.expedia.apiary.extensions.receiver.common.event.ListenerEvent;
+import com.expediagroup.beekeeper.core.model.LifecycleEventType;
+
+import java.util.Locale;
+import java.util.Map;
+
+
+public class IcebergTableListenerEventFilter implements ListenerEventFilter {
+
+  private static final Logger log = LogManager.getLogger(IcebergTableListenerEventFilter.class);
+
+  private static final String FORMAT_KEY = "format";
+  private static final String FORMAT_ICEBERG_VALUE = "ICEBERG/PARQUET";
```

Comment:
What if it's an Iceberg table with a different file format? 
All Iceberg tables should be filtered, regardless of their file format.

Reply:
ah yes, updated filter to look for 'iceberg' only in table format.